makefile

```diff
@@ -4,21 +4,28 @@
 # - Add a line in `.env` with `LOCAL_DEV=true`
 -include .env
 
-ifeq ($(LOCAL_DEV),true)
-	SH=sh -c
-else
-	SH=docker run --rm --user "$(id -u)":"$(id -g)" --volume "${PWD}":/tmp/skip_error --workdir /tmp/skip_error rust:latest sh -c
-endif
+define custom_sh
```

Comment:
I would have call that something like `execute` or `run_cmd` because the `custom` is not really describing anything and the `sh` is collateral to me, it could be something else, what you're looking for is a way to execute a command.